lib/index.js

```diff
@@ -18,13 +18,16 @@ function elevatePoints(points, options) {
             results.push(pt);
         })
         let tp = Cesium.createWorldTerrain();
-        Cesium.sampleTerrainMostDetailed(tp, cpoints)
-        .then((computed) => {
+        var promise = Cesium.sampleTerrainMostDetailed(tp, cpoints);
```

Comment:
Is `var` a deliberate choice here? I'm still trying to figure when is best to use `var` vs `let`

Reply:
No, I don't believe it was.  I know the difference between `var` and `let` in Swift but I'm not sure in JavaScript.  I think I used it in this fashion because I learned `var` when I learned JavaScript in the 90s and that was the last time I had written any JavaScript before now.

Reply:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
Based on that, it should be a `let`.